classes/uGen.sc

```diff
@@ -0,0 +1,20 @@
++ UGen {
+	that { | callback, trigger |
+		var oscAddress = "/that/ugen/%/%".format(synthDef, synthIndex);
```

Comment:
`synthDef` will only add `a SynthDef`. You want some kind of ID, right? But one SynthDef can have many synths. .Why not get the `NodeID.ir`?